### DISCUSSION TITLE
Test Question

### DISCUSSION THREAD

**Discussion by kraih** [mojolicious/mojo#1627] [Q&A] [answered]
Just a test question to see how voting on answers works. So what is your favourite editor, and why is it Vim?

**Comment by Grinnz** [comment]
VSCode obviously.

EDIT: this answer was not given under coercion of our benevolent corporate overlords

**Comment by marcusramberg** [comment]
Doom Emacs. Because it’s vim. 🥇

**Reply by marcusramberg** [reply]
> Doom Emacs. Because it’s vim. 🥇
Also because it’s Evil.

**Comment by jberger** [comment, marked as answer]
Vim because vim

**Reply by davidolrik** [reply]
It's Vim because you can't get out of it ;-)

**Reply by karenetheridge** [reply]
ZZ!

**Comment by janeskil1525** [comment]
IntelliJ IDEA Because of debugging and git integration :-)

**Comment by marcusramberg** [comment]
I just can’t quit you.

ons. 9. des. 2020 kl. 22:30 skrev David Jack Wange Olrik <
notifications@github.com>:

**Comment by yuki-kimoto** [comment]
Hidemaru

**Comment by mche** [comment]
SciTE
